src/OpenLoco/GameState.cpp

```diff
@@ -6,8 +6,15 @@ using namespace OpenLoco::Interop;
 namespace OpenLoco
 {
     loco_global<GameState, 0x00525E18> _gameState;
+    GameStateExtensions _gameStateExtensions;
+
     GameState& getGameState()
     {
         return *_gameState;
     }
+
+    GameStateExtensions& getGameStateExtensions()
```

Comment:
Add mentioned on discord this won't save. Add your variable to the new config as i assume users will want this global rather than per scenario

Reply:
Thinking about it more you probably only want to enable this at certain times. Hmm not sure what would be best.

Reply:
I don't mind about not saving this at the moment.

Reply:
I've looked into this further and I think we can just add another flag to CompanyFlags for this purpose. Then it will even save correctly!

Reply:
done